### DISCUSSION TITLE
Address & Capability Lookup (ACL), including wildcard support

### DISCUSSION THREAD

**Discussion by duke-nuke** [phax/phoss-smp#290] [General]
I am using phoss-smp for the metadata publishing 
our customers are looking for **Address & Capability Lookup (ACL), including wildcard support**
is it already implmented then how can i check this feature ? 
or we have to wait for this feature 

Regards

**Comment by phax** [comment]
Hello @duke-nuke, the Wildcard functionality is supported out of the box with phoss SMP.
The main lookup logic is solely in querying AP, and the SMP has nothing else to do, than to allow Document Type IDs with a `*` in it.
Please see the `SMPClientReadOnly`  class from the peppol-commons project for the wildcard lookup.

Please also note, that there is currently a public review to the Policy for use of Identifiers that will slightly change the semantic of the wildcard lookup mode in the near future.
